pkg/activitypub/store/storeutil/storeutil_test.go

```diff
@@ -7,13 +7,18 @@ SPDX-License-Identifier: Apache-2.0
 package storeutil
 
 import (
+	"fmt"
+	"net/url"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/trustbloc/orb/pkg/activitypub/store/mocks"
 	"github.com/trustbloc/orb/pkg/activitypub/store/spi"
 )
 
+//go:generate counterfeiter -o ../mocks/referenceiterator.gen.go --fake-name ReferenceIterator ../spi ReferenceIterator
```

Comment:
Is this comment supposed to be here? If so, what is it?

Reply:
It generates the mock ReferenceIterator when you run 'go generate ./...'